### ISSUE TITLE
[Feature] Add language support?

### ISSUE THREAD

**Issue by ramazansancar** [murphysecurity/murphysec#111]
### Search before asking

- [X] I had searched in the [issues](https://github.com/murphysecurity/murphysec/issues) and found no similar issues.

### Description

Will you add language support?

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by iseki0** [comment]
What language do you need?

**Comment by iseki0** [comment]
Emmm. Do you means internationalization support? I means... You need language support. Which language? Natural language or Programming language?

**Comment by thediveo** [comment]
I guess this might be a request for better I18N support with at least consistent English language support in the user-facing documentation and especially in the code base. For instance, a quick look into the code base shows hard-coded Chinese language, such as in https://github.com/murphysecurity/murphysec/blob/v3/cmd/murphy/internal/common/view.go:

![hardcoded Chinese user-facing text](https://user-images.githubusercontent.com/6920158/211059122-def7a777-d6fb-4007-a62b-b9b336b0286f.png)

**Comment by ramazansancar** [comment]
Hello @thediveo and @iseki0,
Unfortunately, the main language of the project is Chinese, which makes it difficult for me to understand and develop. If possible, it will be much better to convert parts such as description, readme to English.

**Comment by ramazansancar** [comment]
Yes @thediveo, as you said, if English, the global language of speech and communication, is supported, I would like to offer translation support into my local language, Turkish.

**Comment by iseki0** [comment]
> Hello @thediveo and @iseki0, Unfortunately, the main language of the project is Chinese, which makes it difficult for me to understand and develop. If possible, it will be much better to convert parts such as description, readme to English.

Well, I think it's good. 

@CN-You 你觉得呢

**Comment by iseki0** [comment]
Although the CLI program can be i18n easily, the website and the database are still Chinese only. I will add i18n to the program soon. But after an internal discussion, it looks like we currently have no resources to i18n the database and website. If you have any other suggestions, feel free to tell us.

**Comment by ramazansancar** [comment]
> Although the CLI program can be i18n easily, the website and the database are still Chinese only. I will add i18n to the program soon. But after an internal discussion, it looks like we currently have no resources to i18n the database and website. If you have any other suggestions, feel free to tell us.

There is i18n for Go, in fact, if the default setting is English, I can translate to my native Turkish. You can provide sample areas about database and website, I can think about what we can do about translation.

Thank you for your cooperation

Check this, (GoLang i18n)
- https://github.com/nicksnyder/go-i18n
- https://pkg.go.dev/github.com/nicksnyder/go-i18n/v2/i18n
